test/e2e/specs/undo.test.js

```diff
@@ -10,10 +10,38 @@ import {
 } from '../support/utils';
 
 describe( 'undo', () => {
-	beforeAll( async () => {
+	beforeEach( async () => {
 		await newPost();
 	} );
 
+	it( 'should undo typing after a pause', async () => {
+		await clickBlockAppender();
+
+		await page.keyboard.type( 'before pause' );
+		await new Promise( ( resolve ) => setTimeout( resolve, 1000 ) );
```

Comment:
I wish we could emulate this without _actually_ slowing our test run by one full second.